webrtc-c/canary/src/Cloudwatch.cpp

```diff
@@ -0,0 +1,81 @@
+#include "Include.h"
+
+namespace Canary {
+
+Cloudwatch::Cloudwatch(Canary::PConfig pConfig, ClientConfiguration* pClientConfig)
+    : logs(pConfig, pClientConfig), monitoring(pConfig, pClientConfig), terminated(FALSE)
+{
+}
+
+STATUS Cloudwatch::init(Canary::PConfig pConfig)
+{
+    ENTERS();
+    STATUS retStatus = STATUS_SUCCESS;
+    ClientConfiguration clientConfig;
+    CreateLogGroupRequest createLogGroupRequest;
+    Aws::CloudWatchLogs::Model::CreateLogStreamOutcome createLogStreamOutcome;
+    CreateLogStreamRequest createLogStreamRequest;
+
+    clientConfig.region = pConfig->pRegion;
+    auto& instance = getInstanceImpl(pConfig, &clientConfig);
+
+    if (STATUS_FAILED(instance.logs.init())) {
+        DLOGW("Failed to create Cloudwatch logger, fallback to local output");
+    } else {
+        globalCustomLogPrintFn = logger;
```

Comment:
Can you rename this cloudwatchLogger()? Would like it to be consistent with the producer C canary

Reply:
It sounds awkward to me, since `logger` here is already namespaced by `Cloudwatch`. If we want to get the global symbol (assuming we're using Canary namespace), it would be `Cloudwatch::logger`. If we renamed this to `cloudwatchLogger`, it becomes `Cloudwatch::cloudwatchLogger`. I think It's similar to Producer C's naming, the only difference is that the namespace here is from C++ instead of a manual namespace.

Reply:
Ok. Fair.